src/sentry/search/events/builder.py

```diff
@@ -22,16 +24,19 @@ def __init__(
         orderby: Optional[List[str]] = None,
         auto_aggregations: bool = False,
         use_aggregate_conditions: bool = False,
-        limit: int = 50,
-        offset: Optional[int] = 0,
+        limit: Optional[int] = None,
```

Comment:
[Docs](https://clickhouse.com/docs/en/sql-reference/statements/select/limit-by/) say we can have both `limit` & `limit by`, could we leave the limit=50 default, but let it be optional if you only want limit by (ie. force folks to explicitly say limit=None if they don't want one)